packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -440,6 +441,21 @@ const DataManagerProvider = ({
           initialData.contentType
         );
 
+        const isValidSchema = validateSchema(contentType);
+
+        if (!isValidSchema) {
+          toggleNotification({
```

Comment:
Also, shouldn't this be an error instead of a warning? (just referring to `notification.error...` & the fact this is a form error)

Reply:
We only have 3 different type of notification: warning, info, success.